### ISSUE TITLE
Storybook canvas section is missing landmark and is not accessible via landmark navigation

### ISSUE THREAD

**Issue by micahgodbolt** [storybookjs/storybook#13372]
You can not get to the main section of the storybook site via landmarks

To use landmarks:
1. start narrator
2. press caps lock + pageup/pagedown until you hear 'landmarks'
3. press caps lock + left/right arrow to navigate to each landmark

Note that there is no landmark for the main content

**Comment by micahgodbolt** [comment]
Hah, was going to just post this in our own repo because I found a workaround....but i'll leave it here for ya :D

I was able to solve this by adding a .storybook/preview-body.html with 
```html
<div role="main" id="custom-root"></div>
```

**Comment by adagar** [comment]
I believe this issue was fixed with this PR: https://github.com/storybookjs/storybook/pull/13827

Using VO on Safari, I'm able to see the landmark for `main` content. 

![PXL_20210304_214437121](https://user-images.githubusercontent.com/3698858/110035075-31e1ee80-7d09-11eb-9b96-714f30e0325c.jpg)

Sorry about uploading an image of this quality, I gotta figure out how to take screenshots of VO content
